snuba/datasets/configuration/events_analytics_platform/storages/uptime_monitor_checks.yaml

```diff
@@ -0,0 +1,61 @@
+version: v1
+kind: writable_storage
+name: uptime_monitor_checks
+storage:
+  key: uptime_monitor_checks
+  set_key: uptime_monitor_checks
```

Comment:
I think it should belong to the `events_analytics_platform` since migrations are also link to the `events_analytics_platform` storage set.

I suppose it's not a MUST but since the migrations are already in the `events_analytics_platform` set, I'd keep things together.

Reply:
i've gone ahead and removed the errant uptime_monitor_checks config stuff.